mne/io/base.py

```diff
@@ -50,8 +50,7 @@
 
 
 def _set_pandas_dtype(df, columns, dtype):
-    """Try to set the right columns to categorical"""
-    pandas = _check_pandas_installed(strict=True)
```

Comment:
You don't want to / need to check whether pandas is installed? My suggestion was just to remove the assigment (e.g., change the line `x = y()` to just `y()`) but if we don't need the check here then what you have done is better